Kconfig

```
@@ -20,6 +20,12 @@ config ASTARTE_PAIRING_JWT
 
         Can be obtained with ./generate-astarte-credentials -t pairing -p yourrealmkey.key
 
+config ASTARTE_PAIRING_JWT_MAX_LEN
+    int "Pairing JWT token maximum length"
```

JWT already contains the token word.
```suggestion
    int "Pairing JWT maximum length"
```